Gems/Georeferencing/gem.json

```
@@ -0,0 +1,28 @@
+{
+    "gem_name": "Georeferencing",
+    "version": "1.0.0",
+    "display_name": "Georeferencing",
+    "license": "License used i.e. Apache-2.0 or MIT",
+    "license_url": "https://opensource.org/licenses/Apache-2.0",
+    "origin": "Robotec.ai",
+    "origin_url": "The website for this Gem",
```

(Taken from ROS2 gem.json)

```suggestion
    "origin": "RobotecAI",
    "origin_url": "https://robotec.ai",
```